gef-ui/server/controllers/submit-to-ukef/index.test.js

```diff
@@ -4,6 +4,9 @@ import CONSTANTS from '../../constants';
 
 const { isNotice } = require('../../utils/deal-helpers');
 
+const { RES_MOCK_AIN_APPLICATION_CHECKER } = require('../../utils/MOCKS/mock_applications');
```

Comment:
folders (and files) should not be uppercased (i.e. `MOCKS` should be `mocks`).
This applies here, but it occurs in other places as well so it needs to be updated

Reply:
fixed